hbase-server/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupInfoManagerImpl.java

```diff
@@ -655,22 +663,27 @@ private synchronized void flushConfig(Map<String, RSGroupInfo> newGroupMap) thro
       // according to the inputted newGroupMap (an updated copy of rsGroupMap)
       this.holder = new RSGroupInfoHolder(newGroupMap);
 
+      LOG.debug("New RSGroup map: {}", newGroupMap.toString());
```

Comment:
Its not a good idea to call toString() on the parameter as that defeats the purpose of parameterized logging, if DEBUG is not enabled.  toString() is implicitly evaluated by log4j only if DEBUG logging is enabled for this class. Please get rid of toString() here and below.

https://www.javacodegeeks.com/2015/10/better-performing-non-logging-logger-calls-in-log4j2.html